core/test/utils.test.js

```diff
@@ -0,0 +1,54 @@
+/**
+ @license
+ (C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+import { createNestedObject } from '../utils.js';
+
+/* eslint-disable no-unused-expressions */
+suite('utils', () => {
+  suite('"createNestedObject" function', () => {
+    suite('Parameters Validation', () => {
+      test('Should return undefined when no object or path are provided', () => {
+        expect(createNestedObject(null, null)).to.be.undefined;
+      });
+
+      test('Should return undefined when no path is provided', () => {
+        expect(createNestedObject({}, null)).to.be.undefined;
+      });
+
+      test('Should return undefined when an empty array is provided', () => {
+        expect(createNestedObject({}, [])).to.be.undefined;
+      });
+
+      test('Should return undefined when a different type then object is provided as parameter', () => {
```

Comment:
```suggestion
      test('Should return undefined when a different type than object is provided as parameter', () => {
```

Reply:
nit: I think it's easier to read if we invert different and type:
```suggestion
      test('Should return undefined when a type different than object is provided as parameter', () => {
```